SBBDesignSystemMobileSwiftUI/Views/SBBTabView/TabBarView/TabButtonView.swift

```diff
@@ -0,0 +1,157 @@
+        self.index = index
+        self.contents = contents
+        self.tabBarParameters = tabBarParameters
+        self.isTabBarFocused = isTabBarFocused
+    }
+    
+    public var body: some View {
+        let isPortrait = self.horizontalSizeClass == .compact && self.verticalSizeClass == .regular
+        Button(action: {
+            self.currentTab = self.selectionIndex
+            self.selection = self.contents[index].tag as? Selection ?? self.selection
+            
+            if self.selectionIndex != self.currentTab && !self.isPressed {
+                self.transitionFactor = 0.2
+                withAnimation(Animation.easeIn(duration: 0.2)) {
+                    self.transitionFactor = 1
+                }
+            }
+        }) {
+            if isPortrait {
+                // Display only the icon
+                self.contents[self.index].imageView
+                    .frame(width: self.tabBarParameters.buttonWidth, height: self.tabBarParameters.buttonHeight)
+            } else {
+                // Display the icon and the text
+                HStack(spacing: 0) {
+                    self.contents[self.index].imageView
+                        .frame(width: self.tabBarParameters.buttonWidth, height: self.tabBarParameters.buttonHeight)
+                    
+                    self.contents[self.index].labelView
+                        .background(ViewGeometry())
+                        .onPreferenceChange(ViewSizeKey.self) {
+                            self.labelSizes[self.index] = $0
+                        }
+                        .sbbFont(.body)
+                        .lineLimit(1)
+                        .minimumScaleFactor(0.1)
+                        .padding(.leading, self.selectionIndex == index ? 10 : 0)
+                        .padding(.trailing, 5)
+                }
+                .accessibility(hidden: true)
+            }
+        }
+            .foregroundColor(Color.sbbColor(.textBlack))
+            .simultaneousGesture(
+                DragGesture(minimumDistance: 0)
+                    .onChanged({ _ in
+                        if !self.isPressed {
+                            self.isPressed = true
+                            self.currentTab = self.index
+                            self.transitionFactorPressed = 0
+                            withAnimation(Animation.easeIn(duration: 0.2)) {
+                                self.transitionFactorPressed = 1
+                            }
+                        }
+                    })
+                    .onEnded({ _ in
+                        self.isPressed = false
+                        
+                    })
+            )
+            .accessibility(label: Text(self.isTabBarFocused ? "" : "\("tab bar".localized).") + Text((self.index == self.selectionIndex) ? "\("selected".localized)." : "") + self.contents[self.index].labelView + Text(". \("tab".localized)"))
+            .accessibility(removeTraits: .isButton)
```

Comment:
This line should remove the Button trait, i.e. the VoiceOver should not say 'Button' any more. However, it still does say it.
As of now it says:
  - for the selected tab: Selected. Tab title. Tab. Button. 1/5.
  - for the other ones: Tab title. Tab. Button. 2/5.